From d0a5cb8c01ca49852c45acd28da5bee71456b53d Mon Sep 17 00:00:00 2001
From: Stefan Sauer <ensonic@users.sf.net>
Date: Wed, 14 Dec 2011 10:31:31 +0000
Subject: riff-media: port GST_BUFFER_DATA to 0.11 in conditional code branch

---
diff --git a/gst-libs/gst/riff/riff-media.c b/gst-libs/gst/riff/riff-media.c
index 8fdc695..26da38d 100644
--- a/gst-libs/gst/riff/riff-media.c
+++ b/gst-libs/gst/riff/riff-media.c
@@ -913,14 +913,16 @@ gst_riff_create_video_caps (guint32 codec_fcc,
 
 #if (G_BYTE_ORDER == G_BIG_ENDIAN)
       {
-        guint8 *data = GST_BUFFER_DATA (copy);
+        guint8 *data, *p;
         gint n;
 
+        data = p = gst_buffer_map (copy, NULL, NULL, GST_MAP_READWRITE);
         /* own endianness */
         for (n = 0; n < num_colors; n++) {
-          GST_WRITE_UINT32_BE (data, GST_READ_UINT32_LE (data));
-          data += sizeof (guint32);
+          GST_WRITE_UINT32_BE (p, GST_READ_UINT32_LE (p));
+          p += sizeof (guint32);
         }
+        gst_buffer_unmap (copy, data, -1);
       }
 #endif
       gst_caps_set_simple (caps, "palette_data", GST_TYPE_BUFFER, copy, NULL);
--
cgit v0.9.0.2-2-gbebe
